README.md

```diff
@@ -1,26 +1,44 @@
-# operator-template
+# MySQL Machine Charmed Operator
 
 ## Description
 
-TODO: Describe your charm in a few paragraphs of Markdown
+The [MySQL](https://www.mysql.com/) operator provides an open-source relational database management system (RDBMS). This repository contains a Juju Charm for deploying MySQL on machines.
+
+This charm is currently in development, with High Availability via Group Replication as a short-term goal.
 
 ## Usage
 
-TODO: Provide high-level usage, such as required config or relations
+To deploy this charm using Juju 2.9.0 or later, run:
 
-## Relations
+```shell
+juju add-model gs
+charmcraft pack
+juju deploy ./mysql_ubuntu-20.04-amd64.charm mysql
+```
 
-TODO: Provide any relations which are provided or required by your charm
+To confirm the deployment, you can run:
 
-## OCI Images
+```shell
+juju status --color
+```
 
-TODO: Include a link to the default image your charm uses
+Once MySQL starts up, it will be running on the default port (3306).
 
-## Contributing
+If required, you can remove the deployment completely by running:
+
+```shell
+juju destroy-model -y mysql --destroy-storage
```

Comment:
The model name is previously `gs`

Reply:
That was a typo. Changed in cc37f2fa62a8ccfb339e75b1fd51ab0c8977f1d7